zanata-war/src/main/webapp/WEB-INF/layout/project/languages-list.xhtml

```diff
@@ -70,7 +70,9 @@
   <script type="text/javascript">
   //<![CDATA[
     // refresh checkbox facades on language lists
-    zanata.form.appendCheckboxes(jQuery('##{id}').element);
+    if (typeof zanata != 'undefined') {
```

Comment:
`zanata.form.appendCheckboxes` only needed after the first load, which `zanata` will be available by then.